src/docs/user/references/ScriptLanguageSupport.adoc

```diff
@@ -396,6 +396,50 @@ Or another script binding (for example, USERSPACE):
 my $USERSPACE= $ENV{"USERSPACE"};
 ----
 
+=== PHP
+
+The PHP script engine features the following behaviors:
+
+* Script bindings are translated as environment variables.
+* Workflow variables are un-modifiable except by using a post-script written in another language such as the `update_variables_from_file` script (see <<_bash_and_shell,Bash script language>> for further explanations).
+* PHP script engine is meant to return HTML contents and makes use of link:../user/ProActiveUserGuide.html#_assigning_metadata_to_task_result[result metadata] through the *content.type*="text/html" link:../user/ProActiveUserGuide.html#_generic_information[generic information]. Using the script engine without defining this generic information in the Task is not recommended.
```

Comment:
'HTML content' instead of 'HTML contents'

Reply:
ok